mode-lambda/backend/gl/combine.fragment.glsl

```diff
@@ -56,9 +56,7 @@ void main() {
     if ((! (pz <= 0.0))
         && (0.0 <= ey && ey <= height)
         && (0.0 <= ex && ex <= width)) {
-      fin_Color.rgb =
-          fin_Color.rgb * (1.0 - lay_Color.a)
-        + lay_Color.rgb * lay_Color.a;
+      fin_Color.rgb = fin_Color.rgb * (1.0 - lay_Color.a) + lay_Color.rgb;
```

Comment:
Can you reformat this so only the one line changes, as it is it looks like nothing is different.